### ISSUE TITLE
the meaning of the target_label value(-1)

### ISSUE THREAD

**Issue by tyjiang1997** [sshaoshuai/Pointnet2.PyTorch#18]
Hello, there are three different values (0,1,-1) in the target_label in the loss function DiceLoss(nn.Module); -1 is not considered; I want to know the meaning of these three values It means background, 1 means car, so what does -1 mean?

**Comment by bmankirLinker** [comment]
Those are the ignored values which are set in `DataLoader` for nearby points of the enlarged cuboid as here https://github.com/sshaoshuai/Pointnet2.PyTorch/blob/master/tools/dataset.py#L165

**Comment by tyjiang1997** [comment]
> Those are the ignored values which are set in `DataLoader` for nearby points of the enlarged cuboid as here https://github.com/sshaoshuai/Pointnet2.PyTorch/blob/master/tools/dataset.py#L165

thank you very much
